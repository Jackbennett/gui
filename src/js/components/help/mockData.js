export const helpProps = {
  demoArtifactLink: 'https://dgsbl4vditpls.cloudfront.net/mender-demo-artifact.mender',
  docsVersion: undefined,
  hasMenderShellSupport: true,
  isHosted: false,
  isEnterprise: false,
  menderDebPackageVersion: 'master',
  menderVersion: 'master',
  menderArtifactVersion: 'master',
  org: {
    tenant_token: ''
  }
};
