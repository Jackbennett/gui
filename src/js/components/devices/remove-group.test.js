import React from 'react';
import { render } from '@testing-library/react';
import RemoveGroup from './remove-group';
import { undefineds } from '../../../../tests/mockData';

describe('RemoveGroup Component', () => {
  it('renders correctly', () => {
    const { baseElement } = render(<RemoveGroup onClose={jest.fn} onRemove={jest.fn} />);
    const view = baseElement.getElementsByClassName('MuiDialog-root')[0];
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
