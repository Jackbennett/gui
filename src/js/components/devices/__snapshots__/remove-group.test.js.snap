// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveGroup Component renders correctly 1`] = `
<div
  class="MuiDialog-root"
  role="presentation"
  style="position: fixed; z-index: 1300; right: 0px; bottom: 0px; top: 0px; left: 0px;"
>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  />
  <div
    data-test="sentinelStart"
    tabindex="0"
  />
  <div
    class="MuiDialog-container MuiDialog-scrollPaper"
    role="none presentation"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    tabindex="-1"
  >
    <div
      class="MuiPaper-root MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm MuiPaper-elevation24 MuiPaper-rounded"
      role="dialog"
    >
      <div
        class="MuiDialogTitle-root"
      >
        <h2
          class="MuiTypography-root MuiTypography-h6"
        >
          Remove this group?
        </h2>
      </div>
      <div
        class="MuiDialogContent-root"
      >
        <p>
          This will remove the group from the list. Are you sure you want to continue?
        </p>
      </div>
      <div
        class="MuiDialogActions-root MuiDialogActions-spacing"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text"
          style="margin-right: 10px;"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Cancel
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Remove group
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    data-test="sentinelEnd"
    tabindex="0"
  />
</div>
`;
